src/lib/support/BUILD.gn

```diff
@@ -21,6 +21,9 @@ import("${chip_root}/build/chip/chip_version.gni")
 import("${chip_root}/build/chip/tests.gni")
 import("${chip_root}/src/lib/core/core.gni")
 
+write_file("${root_gen_dir}/include/support/ASN1OID.h",
```

Comment:
This should run at build time via an `action` rule, not graph generation time via `exec_script` and `write_file`.

Generally the only things that need to call `exec_script()` for are those that affect the graph structure or command lines.

We have several examples of code generation already at build time: `buildconfig_header()`, `gdbus_library()`, `gen_flashing_script()`. This case is even simpler than those.

Reply:
Thanks for pointing that out. I've update the script.